Description: Remove root check to be able to test from fakeroot
Author: Andriy Senkovych <jolly_roger@itblog.org.ua>
Last-Update: 2011-03-01

--- a/setup.py
+++ b/setup.py
@@ -84,10 +84,6 @@
         # Mimick the trial script by adding the path as the last arg
         sys.argv.append(test_loc)
 
-        # No superuser should execute tests
-        if hasattr(os, "getuid") and os.getuid() == 0:
-            raise SystemExit('Do not test as a superuser! Exiting ...')
-
         # Add the current dir to path and pull it all together
         sys.path.insert(0, os.path.curdir)
         sys.path[:] = map(os.path.abspath, sys.path)
